app/components/Inputs/Label/Label.jsx

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react'
+import classNames from 'classnames'
+import { omit } from 'lodash-es'
+
+import RefreshIcon from '../../../assets/icons/refresh.svg'
+import styles from './Label.scss'
+
+type Props = {
+  label: string,
+  loading: boolean,
+  labelClassName: string,
+  labelContainerClassName: string,
+  renderAdditionalContent: () => React$Node
+}
+
+export default class Label extends React.Component<Props> {
+  render() {
+    const { label, loading, renderAdditionalContent } = this.props
+
+    const passDownProps = omit(
+      this.props,
+      'label',
+      'loading',
+      'labelClassName',
+      'labelContainerClassName',
+      'renderAdditionalContent'
+    )
```

Comment:
This `omit` method is used across the app but it not ideal - requires any future changes to props to be mirrored and also perhaps needlessly verbose.

Wdyt of this instead?:
```js
const {
  label,
  loading,
  labelClassName,
  labelContainerClassName,
  renderAdditionalContent,
  ...passDownProps // <-- rest of props
} = this.props

const labelClassName = classNames(styles.label, labelClassName)
const labelContainerClassName = classNames(styles.labelContainer, labelContainerClassName)
```
This way props used in this class will automatically be omitted from the nested components.

Reply:
Haha this is a much cleaner solution that then the pattern used throughout the app thanks @ranbena! Will update.